corehq/apps/cloudcare/static/cloudcare/js/form_entry/form_ui.js

```diff
@@ -619,9 +633,29 @@ hqDefine("cloudcare/js/form_entry/form_ui", [
                 }
                 element.pendingAnswer(constants.NO_PENDING_ANSWER);
             } else {
-                response.children = response.tree;
+                const allChildren = response.tree;
                 delete response.tree;
+
+                if (deletedGroup) {
+                    const ixParts = deletedGroup.split(",");
+                    response.children = JSON.parse(JSON.stringify(allChildren));
```

Comment:
What's the purpose of this?

Reply:
Making a copy of that Json object because it gets changed during the ko.mapping update. I see now that underscore has cloneDeep. I'll use that instead.